eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```diff
@@ -41,6 +67,8 @@ jobs:
         - task: Powershell@2
           inputs:
             pwsh: true
+            ${{ if parameters.SparseCheckout }}:
+              workingDirectory: $(Build.Repository.LocalPath)
```

Comment:
Instead of overriding this here should we pass in a path to the sparse-checkout template and make it the same as the default checkout?

Reply:
I'll make it more consistent.

Reply:
Looks like it works fine since everything defaults there anyway. This was a holdover from when I was doing `git clone` into a sub-directory instead of to `.`.